src/utils.tsx

```diff
@@ -69,15 +76,39 @@ const toPercentage = (val: number) => Math.round(val * 100) + '%'
 
 const toPrettyNumber = (val: number) => Math.round(val * 10) / 10
 
+const eventTypeNames = new Map<number, string>([
+  [0, ''],
+  [1, ''],
+  [5, 'DCPD'],
+  [2, 'DCP'],
+  [3, 'CPD'],
+  [4, 'CPF'],
```

Comment:
We don't need labels for championship. If you really want labels, can it be CMP for both?